editor/components/block-list/index.js

```diff
@@ -37,6 +37,15 @@ import {
 } from '../../selectors';
 import { startMultiSelect, stopMultiSelect, multiSelect, selectBlock } from '../../actions';
 
+function isAnyInputFieldActive() {
```

Comment:
I guess we could also do something like `isInputField( document.activeElement )`, but for me it's either.